lib/rspec/core/minitest_assertions_adapter.rb

```diff
@@ -1,9 +1,9 @@
 begin
-  # Only the minitest 5.x gem includes the minitest.rb and assertions.rb files.
+  # Only Minitest 5.x includes minitest.rb and assertions.rb files
   require 'minitest'
   require 'minitest/assertions'
 rescue LoadError
-  # We must be using Ruby Core's MiniTest or the Minitest gem 4.x.
+  # We must be using Minitest 4.x
```

Comment:
Restore these, it explains the reasoning better...

Reply:
There's no Ruby Core (stdlib?) MiniTest in Ruby 2.3+.